public/app/percona/backup/components/BackupInventory/AddBackupModal/AddBackupModal.service.ts

```diff
@@ -7,14 +7,23 @@ import { SelectableService } from './AddBackupModal.types';
 export const AddBackupModalService = {
   async loadServiceOptions(): Promise<Array<SelectableValue<SelectableService>>> {
     let result: Array<SelectableValue<SelectableService>> = [];
+    const supportedServices: Databases[] = [Databases.mysql, Databases.mongodb];
     const services = await InventoryService.getDbServices();
 
-    // TODO remove this constraint when more DB types are supported
-    if (services.mysql) {
-      result = services.mysql.map(
-        ({ id, name }): SelectableValue<SelectableService> => ({ label: name, value: { id, vendor: Databases.mysql } })
-      );
-    }
+    Object.keys(services).forEach((serviceName: Databases) => {
+      if (supportedServices.includes(serviceName) && services[serviceName]) {
+        const newServices = services[serviceName] || [];
```

Comment:
nit: use `??` instead of `||`, if you're just using it as a fallback for `undefined` or `null`

Reply:
👍